docs/demo/demo_single_sys_plotter.py

```diff
@@ -44,10 +44,10 @@ def main():
     
     ### Plot radius attributes 
     axes[0][1].plot(df['Time'], df['SemiMajorAxis'], linestyle='-', c='k', label='Semi-Major Axis')
-    axes[0][1].plot(df['Time'], df['Radius_1']/df['Radius_1/RL'], linestyle='--', alpha=0.5, c='r', label='Roche Radius 1')
-    axes[0][1].plot(df['Time'], df['Radius_2']/df['Radius_2/RL'], linestyle='--', alpha=0.5, c='b', label='Roche Radius 2')
-    axes[0][1].plot(df['Time'], df['Radius_1'], linestyle='-', c='r', label='Stellar Radius 1')
-    axes[0][1].plot(df['Time'], df['Radius_2'], linestyle='-', c='b', label='Stellar Radius 2')
+    axes[0][1].plot(df['Time'], df['Radius(1)']/df['Radius_1/RL'], linestyle='--', alpha=0.5, c='r', label='Roche Radius 1')
```

Comment:
Does Radius_1/RL need to be changed? If not here, then in COMPAS?

Reply:
No to your first question, and yes to the second question. But changing COMPAS headers is beyond the scope of this pull request.

Reply:
Why does it need to be changed?  If you need the RL radius, you can get it as 
'RocheLobe_1/a' * 'Separation' * (1-'Eccentricity'),
[or, as @themikelau has done, as 'Radius_1' / 'Radius_1/RL' ]
so there are already two ways of retrieving it in post-processing...

Reply:
@ilyamandel: I think @reinhold-willcox was referring to whether the header name 'Radius_1/RL' has to be similarly changed to 'Radius(1)/RL' as I have done with the other header names, to reflect the updated convention in primary/secondary designation. I was then saying it is not necessary because the previous commits that made these changes in header names forgot to change 'Radius_1/RL' (and 'Radius_2/RL')

Reply:
Yes, I'll bring in @jeffriley as well, since I noticed a few weeks ago that the new suffix changes, e.g (1), have not been applied everywhere. Jeff said he would fix those in a future PR: Jeff will that include the changes to this variable as well?